README.txt

```diff
@@ -8,6 +8,8 @@ This library also provides a command line interface for offline translation of s
 
 For SPIRV translation, this library depends on SPIRV-Cross: https://github.com/KhronosGroup/SPIRV-Cross
 For compiling to DXIL, dxcompiler.dll and dxil.dll are required (or your platform's equivalent).
+These dependencies can be obtained in the Vulkan SDK: https://vulkan.lunarg.com/
```

Comment:
As far as I can tell, the Vulkan SDK for Windows doesn't include dxil.dll
Maybe one should use https://github.com/microsoft/DirectXShaderCompiler/releases instead?

Reply:
You're totally right, I must have put that dll in my own VulkanSDK dir. I'll add the link to the DXC release page.